### ISSUE TITLE
[BUG] Blank MLflow UI on Windows – static JS served as text/plain in v3.5.1

### ISSUE THREAD

**Issue by Rahul20037237** [mlflow/mlflow#18620]
### Issues Policy acknowledgement

- [x] I have read and agree to submit bug reports in accordance with the [issues policy](https://www.github.com/mlflow/mlflow/blob/master/ISSUE_POLICY.md)

### Where did you encounter this bug?

Local machine

### MLflow version

- Client: 1.x.y
- Tracking server: 1.x.y

### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **Python version**:
- **yarn version, if running the dev UI**:

### Describe the problem

 MLflow UI serves static files with wrong MIME type (text/plain) on localhost

I tried running a simple MLflow tracking program using version 3.5.1.
The program executed successfully, and the MLflow server started without any visible errors.
However, when I opened the MLflow UI in the browser (via the provided local port link), the page loaded as a blank screen.
Upon inspecting the browser console, I found an error stating that the JavaScript files were being served with the incorrect MIME type (text/plain), which prevented them from executing.
This issue suggests that the MLflow UI’s static file serving configuration on Windows may not be correctly setting MIME types for JavaScript assets. and  tried with different browser still, i got a same error. and also i enabled a  JavaScript permission
 
 mlflow.set_tracking_uri(uri="http://127.0.0.1:5000")

 mlflow.set_experiment("MLflow Quickstart")

with mlflow.start_run():
 
    mlflow.log_params(params)

    mlflow.log_metric("accuracy", accuracy)

    mlflow.set_tag("Training Info", "Basic LR model for iris data")

    signature = infer_signature(X_train, lr.predict(X_train))

    model_info = mlflow.sklearn.log_model(
        sk_model=lr,
        artifact_path="iris_model",
        signature=signature,
        input_example=X_train,
        registered_model_name="tracking-quickstart",
    )

<img width="837" height="489" alt="Image" src="https://github.com/user-attachments/assets/394b27fe-ca2e-4084-b481-e6bfd339b900" />

<img width="1541" height="125" alt="Image" src="https://github.com/user-attachments/assets/9548eec7-cede-4992-adc0-83032fa815cf" />

### Tracking information

```shell
REPLACE_ME
```

### Code to reproduce issue

```
REPLACE_ME
```

### Stack trace

```
REPLACE_ME
```

### Other info / logs

```
REPLACE_ME
```

### What component(s) does this bug affect?

- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/scoring`: MLflow model serving, deployment tools, Spark UDFs
- [ ] `area/evaluation`: MLflow model evaluation features, evaluation metrics, and evaluation workflows
- [ ] `area/prompt`: MLflow prompt engineering features, prompt templates, and prompt management
- [ ] `area/tracing`: MLflow Tracing features, tracing APIs, and LLM tracing functionality
- [ ] `area/gateway`: MLflow AI Gateway client APIs, server, and third-party integrations
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/uiux`: Front-end, user experience, plotting
- [ ] `area/docs`: MLflow documentation pages

**Comment by harupy** [comment]
HTML source:

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">
    <link rel="shortcut icon" href="./static-files/favicon.ico">
    <meta name="theme-color" content="#000000">
    <link rel="manifest" href="./static-files/manifest.json" crossorigin="use-credentials">
    <title>MLflow</title>
    <script defer="defer" src="static-files/static/js/main.03f3f89d.js"></script>
    <link href="static-files/static/css/main.280d6c90.css" rel="stylesheet">
</head>
<body>
    <noscript>You need to enable JavaScript to run this app.</noscript>
    <div id="root" class="mlflow-ui-container"></div>
    <div id="modal" class="mlflow-ui-container"></div>
</body>
</html>
```

Error

```
The resource from “http://127.0.0.1:5000/static-files/static/js/main.03f3f89d.js” was blocked due to MIME type (“text/plain”) mismatch (X-Content-Type-Options: nosniff). [Learn More]
Loading failed for the <script> with source “http://127.0.0.1:5000/static-files/static/js/main.03f3f89d.js”.
```

**Comment by harupy** [comment]
Found https://www.taricorp.net/2020/windows-mime-pitfalls/

**Comment by harupy** [comment]
https://github.com/pallets/flask/issues/1045#issuecomment-42202749 seems related

**Comment by harupy** [comment]
@Rahul20037237 can you open regedit and check the `HKEY_CLASSES_ROOT` key as suggested in https://github.com/pallets/flask/issues/1045#issuecomment-42202749.

**Comment by harupy** [comment]
Filed https://github.com/mlflow/mlflow/pull/18647 but I'll close it if fixing the registry value works since adding MIME types might have undesired side effects.

**Comment by Rahul20037237** [comment]
Root cause identified as an incorrect MIME type mapping in the Windows registry (text/plain for .js). After changing it to application/javascript, the MLflow UI loaded correctly. Closing this as resolved.
